src/OrchardCore.Modules/OrchardCore.Seo/Drivers/SeoMetaPartDisplayDriver.cs

```diff
@@ -0,0 +1,161 @@
+                {
+                    model.OpenGraphType = part.OpenGraphType;
+                    model.OpenGraphTitle = part.OpenGraphTitle;
+                    model.OpenGraphDescription = part.OpenGraphDescription;
+                    model.SeoMetaPart = part;
+                }).Location("Parts#Seo;50%Open Graph;20"));
+            }
+
+            if (settings.DisplayTwitter)
+            {
+                results.Add(Initialize<SeoMetaPartTwitterViewModel>("SeoMetaPartTwitter_Edit", model =>
+                {
+                    model.TwitterTitle = part.TwitterTitle;
+                    model.TwitterDescription = part.TwitterDescription;
+                    model.TwitterCard = part.TwitterCard;
+                    model.TwitterCreator = part.TwitterCreator;
+                    model.TwitterSite = part.TwitterSite;
+                    model.SeoMetaPart = part;
+                }).Location("Parts#Seo;50%Twitter;30"));
+            }
+
+            if (settings.DisplayGoogleSchema)
+            {
+                results.Add(Initialize<SeoMetaPartGoogleSchemaViewModel>("SeoMetaPartGoogleSchema_Edit", model =>
+                    {
+                        model.GoogleSchema = part.GoogleSchema;
+                        model.SeoMetaPart = part;
+                    }).Location("Parts#Seo;50%Google Schema;40"));
+            }
+
+            return Combine(results);
+        }
+
+        public override async Task<IDisplayResult> UpdateAsync(SeoMetaPart part, IUpdateModel updater)
+        {
+            var partViewModel = new SeoMetaPartViewModel();
+            if (await updater.TryUpdateModelAsync(partViewModel, Prefix))
+            {
+                try
+                {
+                    part.Render = partViewModel.Render;
+                    part.PageTitle = partViewModel.PageTitle;
+                    part.MetaDescription = partViewModel.MetaDescription;
+                    part.MetaKeywords = partViewModel.MetaKeywords;
+                    part.Canonical = partViewModel.Canonical;
+                    part.MetaRobots = partViewModel.MetaRobots;
+
+                    part.CustomMetaTags = String.IsNullOrWhiteSpace(partViewModel.CustomMetaTags)
+                        ? Array.Empty<MetaEntry>()
+                        : JsonConvert.DeserializeObject<MetaEntry[]>(partViewModel.CustomMetaTags);
+
+                    if (part.Canonical?.IndexOfAny(SeoMetaPart.InvalidCharactersForCanoncial) > -1 || part.Canonical?.IndexOf(' ') > -1)
+                    {
+                        updater.ModelState.AddModelError(Prefix, S["The canonical entry contains invalid characters."]);
+                    }
+                }
+                catch
+                {
+                    updater.ModelState.AddModelError(Prefix, S["The meta entries are written in an incorrect format."]);
+                }
+            }
+
+            var openGraphModel = new SeoMetaPartOpenGraphViewModel();
```

Comment:
I didn't make the check because I was expecting that if you don't provide the values, they will be set to empty once published, which is what I expect if I decide to not display the inputs/panels. But I might be wrong.